DataFormats/PatCandidates/interface/MET.h

```diff
@@ -200,11 +247,25 @@ namespace pat {
       // ---- holder for pfMET specific info ---
       std::vector<SpecificPFMETData> pfMET_;
 
+      // MET significance
+      double metSig_;
+      
+      const PackedMETUncertainty findMETTotalShift(MET::METCorrectionLevel cor, MET::METUncertainty shift) const;
+   
+      std::map<MET::METCorrectionLevel, std::vector<MET::METCorrectionType> > corMap_;
+      void initCorMap();
+
+    
+
     protected:
 
       // ---- non-public correction utilities ----
+      //kept for 74X backward-compatibility, not initialized and used in 75X
```

Comment:
My comment for the format change still applies: if the old 3 vectors map to values in the new uncertainties_ vector, you need to define an iorule in classes_def.xml file and remove the old 3 vectors.